doc/prediction.rst

```diff
@@ -146,3 +146,48 @@ instance we might accidentally call ``clf.set_params()`` inside a predict functi
 
     with ThreadPoolExecutor(max_workers=10) as e:
         e.submit(predict_fn, ...)
+
+*****************************
+Privacy-Preserving Prediction
+*****************************
+
+Concrete ML is a third-party open-source library developed by `Zama`_ that proposes gradient
```

Comment:
I'd put the link to `Concrete ML` here instead of the last sentence.

Reply:
You're right, thx